tests/unit/utils/factory_unittest.cc

```diff
@@ -54,7 +55,7 @@ TEST(Factory, DISABLED_string_features_from_file)
 	file_save->close();
 
 	auto file_load = std::make_shared<CSVFile>(filename.c_str(), 'r');
-	auto obj = string_features(file_load, DNA, PT_CHAR);
+	auto obj = create_string_features(file_load, DNA, PT_CHAR);
```

Comment:
I am a bit confused why we still have python style methods in c++ though :)

Reply:
i forget to delete those codes, will repair in #5011.
https://github.com/shogun-toolbox/shogun/blob/376bf8f500d3856f5ffe6f145dff84cbf55de104/src/shogun/util/factory.h#L382-L402